configuration/items.md

```diff
@@ -298,8 +300,8 @@ Please refer to the article on [Transformations](transform.html) for more usage
 ### Icons
 
 The icon name is used by openHAB to select the image to display next to an Item name when using one of openHAB's UIs, e.g. Basic UI.
-The icon name appears between angle brackets "<>".
-In the example below, the "switch" icon has been selected for use with the Item named, "Livingroom Ceiling Light":
+The icon name appears between angle brackets "&lt;&gt;".
+In the example below, the "switch" icon has been selected:
```

Comment:
Markdown would otherwise remove the `<>` characters
The second part of the sentence seemed redundant